### ISSUE TITLE
Support query openjdk bug tracker to get tracker status

### ISSUE THREAD

**Issue by sophia-guo** [adoptium/aqa-tests#2907]
We also exclude tests by [openjdk bug tracker](https://bugs.openjdk.java.net/projects/JDK/issues/JDK-8274381?filter=allopenissues) would prefer to query [openjdk bug tracker](https://bugs.openjdk.java.net/projects/JDK/issues/JDK-8274381?filter=allopenissues) status when updating [ProblemList.json](https://github.com/adoptium/aqa-tests/blob/master/disabledTestParser/ProblemList.json)

In [issue_status.py](https://github.com/adoptium/aqa-tests/blob/master/disabledTestParser/issue_status.py), when query we need to figure out the ISSUE_TRACKER type first and query the status correspondingly.

**Comment by pacificPlumeria** [comment]
I can try this out 😊

**Comment by smlambert** [comment]
Thanks @pacificPlumeria !  If you have questions, don't hesitate to ask (either in this issue or in Slack)!

**Comment by pacificPlumeria** [comment]
Im sorry im not sure i grasp what need to be done!

**Comment by smlambert** [comment]
Essentially my understanding of this one is that the python script [issue_status.py](https://github.com/adoptium/aqa-tests/blob/master/disabledTestParser/issue_status.py) reads ProblemList.txt files (found in this excludes directory).  There are 2 types of Issue URLs, one for github issues, and one called JBS at https://bugs.openjdk.java.net/.

If you look at an example ProblemList file, at these 2 entries found at: https://github.com/adoptium/aqa-tests/blob/master/openjdk/excludes/ProblemList_openjdk17.txt#L28-L29

```
java/beans/PropertyEditor/TestFontClassValue.java	https://bugs.openjdk.java.net/browse/JDK-8173082 macosx-all
java/beans/XMLEncoder/java_awt_CardLayout.java	https://github.com/adoptium/aqa-tests/issues/2810 aix-all
```
one of the excludes is tracked by an issue in JBS, one is tracked by an issue in github.  I believe @sophia-guo is suggesting to update the python script to check what type an issue is, if an issue is a JBS one or a github one 



@sophia-guo - can you please add some more specific direction in this issue to clarify what you want addressed?

**Comment by MenusGarg5981** [comment]
Hey @sophia-guo , I want to take up this issue.

**Comment by smlambert** [comment]
Un-assigning this issue and putting this back into the set of open / available good first issues.

**Comment by smlambert** [comment]
Actually moving this one to the canosp project list as it relates to the "Disabled tests pipeline" work.

**Comment by gervaisj** [comment]
@smlambert Oops, I did not see this issue. I believe this was resolved by https://github.com/adoptium/aqa-tests/pull/3416

**Comment by smlambert** [comment]
Closed via #3416.
